pegasus/forms/hoc_survey_2014.rb

```diff
@@ -1,189 +0,0 @@
-class HocSurvey2014
```

Comment:
do we need to delete this or could we leave it? It might be useful in case we need to reindex HocSurvey2014 forms for some reason